src/tests/ftest/launch.py

```diff
@@ -556,25 +556,28 @@ def get_test_list(tags):
     return test_tags, test_list
 
 
-def get_test_files(test_list, args, yaml_dir):
+def get_test_files(test_list, args, yaml_dir, vmd_flag=False):
     """Get a list of the test scripts to run and their yaml files.
 
     Args:
         test_list (list): list of test scripts to run
         args (argparse.Namespace): command line arguments for this program
         yaml_dir (str): directory in which to write the modified yaml files
+        vmd_flag (bool): PCI address list contains VMD address.
 
     Returns:
         list: a list of dictionaries of each test script and yaml file; If
             there was an issue replacing a yaml host placeholder the yaml
             dictionary entry will be set to None.
 
     """
-    test_files = [{"py": test, "yaml": None} for test in test_list]
+    test_files = [{"py": test, "yaml": None, "env": {}} for test in test_list]
     for test_file in test_files:
         base, _ = os.path.splitext(test_file["py"])
-        test_file["yaml"] = replace_yaml_file(
-            "{}.yaml".format(base), args, yaml_dir)
+        yanl_file, env_vars = replace_yaml_file(
```

Comment:
nit: typo, but at least it's consistent. ;)

Reply:
Sure. Will make sure it is fixed on some PR. Thanks.